networkit/cpp/distance/test/DynSSSPGTest.cpp

```diff
@@ -112,6 +150,47 @@ TEST_F(DynSSSPGTest, testDynamicBFS_batch) {
 
 }
 
+TEST_F(DynSSSPGTest, testDynamicBFS_batchDeletion) {
+/* Graph:
+        0    3   6
+        \  / \ /
+          2 - 5
+        /  \ / \
+        1    4   7
+*/
+    count n = 9;
+    Graph G(n);
```

Comment:
```suggestion
    Graph G(9);
```
Also, the graph above has 8 nodes, not 9.